tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932237.yaml

```
@@ -209,3 +209,19 @@ tests:
             version: HTTP/1.0
           output:
             no_log_contains: id "932237"
+  - test_title: 932237-14
+    desc: "FP with word settings Github Issue #3393"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              User-Agent: "https://example.com/?args=settings"
+              Host: localhost
+              Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+            method: GET
+            port: 80
+            uri: /
```

Uset HTTP verbs in URIs (improves performance with go-ftw and httpbin) (won't repeat for the other locations):
```suggestion
            uri: /get
```

---

not sure I understand what you mean, do you want me to use a URI path other than root or do you want me to set it to the same as the request method i.e ``/get`` or ``/post``?

---

To the same verb as the request method. See https://coreruleset.org/docs/development/contribution_guidelines/#using-the-correct-http-endpoint